content/en/software/install-and-run/running-on-linux.md

```diff
@@ -19,10 +19,10 @@ Medley can be installed on your Linux system in one of two configurations: *stan
 
 ## **Standard Installation \(Debian-based systems only\)**
 
-We are currently only building standard installations for Debian-based systems (i.e., systems that support dpkg).
+We are currently only building standard installations for Debian-based systems (i.e., systems that support dpkg). If you need something else, let us know.
```

Comment:
Should this be submit an issue?

Reply:
I agree.  Can we provide a clean link to the issue submission github page?

Reply:
i took back out the offending sentence. In general  we're not offering support or any particular enhancements, or even promising we'll accept things that don't meet our standards.